Exercise_B/index.css

```diff
@@ -2,3 +2,24 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.square{
+    width: 60px;
+    height: 60px;
+    margin: 2px;
+}
+
+.a{
+    background-color: blue;
+    float: left;
+}
+.b{
+    background-color: red;
+    position: relative;
+    top: 100px;
```

Comment:
You do not need to use top and left.
Its possible to use the properties
clear:
margin:auto

consider making a,b,c id's as they are unique.


Well done for implementing the website in your own way~